flink-runtime/src/test/java/org/apache/flink/runtime/rest/RestEndpointITCase.java

```diff
@@ -72,14 +72,14 @@ public void testEndpoints() throws Exception {
 		RestServerEndpointConfiguration serverConfig = RestServerEndpointConfiguration.fromConfiguration(config);
 		RestClientConfiguration clientConfig = RestClientConfiguration.fromConfiguration(config);
 
-		final String address = "localhost";
```

Comment:
it appears this change belongs into another commit.

Reply:
Sorry, this must be a rebasing artifact. Will remove it.